.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: ci
+
+on:
+  - push
```

Comment:
Would run on push to any branch.

Reply:
That may be fine on such low-traffic repo.  Either way works for me

Reply:
Changed just in case.